### PR TITLE
chore(production): bump version of db-migrations

### PR DESCRIPTION
**Pull request by AndreasHuber**

### CODE REVIEW

File: envs/production/api.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ locals {
     image_tags = {
       database_layer   = "0.3.78"
       server           = "production"
-      api_db_migration = "0.18.0"
+      api_db_migration = "0.20.0"
```
Comment (AndreasHuber) [inline envs/production/api.tf:6]: Does this contain another migration - apart from adding a description - that we do not want to include?

**Comment by AndreasHuber** [comment]
OpenAI key [is already added](https://github.com/serlo/infra/pull/32/files) for both staging and production, right?

File: envs/production/api.tf (hunk 1 above)
Comment (AndreasHuber) [reply envs/production/api.tf:6]: The [migration 0.19.0](https://github.com/serlo/db-migrations/pull/327/files) would create an exercise_submission table. This does not seem dangerous to me but maybe give it a short look before deploying @hugotiburtino.

**Comment by hugotiburtino** [comment]
> OpenAI key [is already added](https://github.com/serlo/infra/pull/32/files) for both staging and production, right?

yes, thanks for asking

**Pull request merged**